app/main.py

```diff
@@ -1,6 +1,17 @@
 from typing import Callable
+from typing import Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    seve_res = {}
+
+    def wrapper(*args) -> Any:
+        if args in seve_res:
```

Comment:
what if two different functions have the same `args` but different `kwargs`?
you must use `args` + `kwargs` as the keys of `func_cache`